pkg/apis/networking/validation/validation.go

```diff
@@ -99,6 +152,8 @@ func ValidateNetworkPolicyUpdate(update, old *networking.NetworkPolicy) field.Er
 	return allErrs
 }
 
+// ValidatePolicyTypes validates the PolicyTypes field in a NetworkPolicySpec
```

Comment:
Is this missing a function body to go with the comment?

Reply:
I think this is a mis-type. I was going to split this out in the future.